src/controllers/requestController.js

```diff
@@ -82,5 +85,44 @@ class Requests {
       return Response.errorResponse(res, 500, 'Something went wrong', error);
     }
   }
+
+  /**
+   * @param {object} req request
+   * @param {object} res response
+   * @return {function} requests
+   */
+  async EditRequest(req, res) {
+    try {
+      const rawData = req.body;
+      const { id } = req.params;
+      // check if the object is empty
+      if (Helpers.isEmpty(rawData)) {
+        return Response.customResponse(res, 400, "Can't send an empty object");
+      }
+
+      // update the object
+      let data = await requestService.updateRequest(rawData, id);
+      const roleDetails = await userService.findUserByRole('Manager');
+      const {
+        from, travelDate, returnDate, reason, status, updatedAt
+      } = data.dataValues;
+      data = {
+        from,
+        travelDate,
+        returnDate,
+        reason,
+        status,
+        updatedAt,
+        manager: roleDetails.dataValues.userEmail,
+        user: req.user.firstName,
+        userEmail: req.user
+      };
```

Comment:
I think theres a typo. 
`userEmail: req.user.userEmail`

Reply:
I have removed that